topics/earth-system/tutorials/ocean-data-view/tutorial.md

```diff
@@ -0,0 +1,249 @@
+> 2. Select **1 SCATTER Window** another tab opens (you now have 3 of them)
+> ![Image of the new open tab for the preparation of a scatter view](../../images/coastal_water_dyn/scatter_prep.png)
+> 1. On the window click on the right side of your mousse
+> 4. Select **X-Variable**, press **TEMP [degree_Celsius]** and OK.
+> 5. Select **Y-Variable**, press **PRES [decibar]**, at the bottom of the pop-up check the box "Reverse range" and OK.
+> ![Selection of the Pression and check of the Reverse range button on the pop-up image](../../images/coastal_water_dyn/yvar_pres.png)
+> You now have the final image with all the curves for each stationside by side. You can navigate in them by using the arrows left/right on your keyboard with the red curve being the station you're on. 
+> ![Resulting image of the scatter plot](../../images/coastal_water_dyn/final_scatter.png)
+{: .hands_on}
+
+> <hands-on-title>Save your analysis</hands-on-title>
+> 1. Click right on the scatter image and select **Save plot as**
+> 2. In the pop-up select and go in the **outputs** folder
+> ![Image of the pop-up showing the outputs folder to go in](../../images/coastal_water_dyn/outputs.png)
+> 3. Once you're in the folder you can rename your scatter plot in "File Name:" by "1900392_prof_scatter"
+> 4. You can also change the "Files of type:" by "PNG (*.png *.PNG) 
+> 5. Click on **Save**, then OK and OK.
+> 6. On the map click right and then **Save Map As...**
+> 7. Still in the outputs folder, rename it "1900392_prof_map" check that the "Files of type:" is "PNG (*.png *.PNG) 
+> 8. Click on **Save**, then OK and OK
+>
+> Now, if you have finished with your analysis you can exit ODV. To do so you need to do it properly. 
+>
+> 9. On th top left click on **File** select **Exit**
+> 10. If you want to save the other window also click on **Yes**. Here we don't need it so click **No**.
+>
+> You can now go back on your Galaxy instance.
+{: .hands_on}
+
+Now, after waiting for everything to turn green in your history, you can see 2 new outputs "1900392_prof_scatter" and "1900392_prof_map"
+
+> <hands-on-title>View in Galaxy</hands-on-title>
+> In the history panel click on the {% icon galaxy-eye %} (eye) icon of your output.
+>
+> You can now visualize the outputs in galaxy middle pannel. 
+{: .hands_on}
+
+![Image in the middle pannel of "1900392_prof_scatter"](../../images/coastal_water_dyn/scatter.png)
+![Image in the middle pannel of "1900392_prof_map"](../../images/coastal_water_dyn/map.png)
+
+
+# Conclusion
+
+Great you now know how to use argo floater data to create profiles curves in Ocean Data View (ODV) within Galaxy.
+
+# Extra information
+Coming up soon follow ups tutorials on Coastal Water Dynamics workflow. Keep an {% icon galaxy-eye %} open if you are interested !
+Theese tutorials are made within the [EOSC FAIR-EASE project](https://fairease.eu/).
+![Project funded by the european union](../../images/coastal_water_dyn/eu_fe.png)
```

Comment:
These should be recored as a 'funding' key in the `contributions:` E.g.:

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - fairease
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

You'll need to add a contributor with the right name to CONTRIBUTORS.yaml